osu.Game/Screens/Select/Carousel/CarouselBeatmapSet.cs

```diff
@@ -122,7 +129,19 @@ private int compareUsingAggregateMax(CarouselBeatmapSet other, Func<BeatmapInfo,
         public override void Filter(FilterCriteria criteria)
         {
             base.Filter(criteria);
-            Filtered.Value = Items.All(i => i.Filtered.Value);
+            bool match = Items.All(i => i.Filtered.Value);
+
+            if (BeatmapSet?.Equals(criteria.SelectedBeatmapSet) == true)
```

Comment:
Where does this come from?

Reply:
Please don't self-resolve code reviews (we will do this once the issue is resolved).

Can you explain the intention behind this addition? I'm curious where it came from.

Reply:
If you're going to go out of your way to re-request a review from me please at least read my comments and provide answers.

Reply:
Sorry.. I didn't see the second comment when I re-requested since this comment was unexpanded. I didn't mean to ignore your message.

I added that piece of code since it looked like an important checker that was needed from the code I referenced from, but I realize now that its not needed at all since the user isn't selecting a beatmap at all.

Sorry again for ignoring your message.